pages/hub/apps/guides/custom-markets.mdx

```diff
@@ -0,0 +1,51 @@
+import { Callout } from 'components'
+
+The Azuro Protocol also offers markets for world events, such as the "2024 Presidential Election USA." 
+
+<Callout type="info">
+  If you use our [SDK](/hub/apps/sdk/overview), no additional changes are needed; just ensure you are using version 5.1.0 or above.
+</Callout>
+
+The steps to obtain custom markets are the same as for sports markets, but there is a slight difference in how the names of markets and selections are generated.
```

Comment:
The steps to get custom (i.e. markets other than sport markets) are the same....

Reply:
ed713387326b0f5d2cc3aa32c1ea95d8f69953b4